README.md

```diff
@@ -250,6 +250,8 @@ This is a comma-separated list of the following options:
   [FlameGraph](https://github.com/brendangregg/FlameGraph) script. This is
   a collection of call stacks, where each line is a semicolon separated list
   of frames followed by a counter.
+  - `calltree[=C]`  - produce call tree in HTML format. C is counter type: 'samples' or 'total'.  
+  - `backtrace[=C]` - produce backtrace tree HTML format. C is counter type: 'samples' or 'total'.   
```

Comment:
Why two different options? One "tree" with optional "--reverse" (as was before) looks cleaner and more logical.
It does not work anyway if two options specified at the same time.
Since there is already too many dump options, I'm thinking about changing the set of booleans to one enum.

Reply:
Can please revisit this. We want to have both calltree and backtrace in every run.  Users can select what they want.

Backtrace is important for analysis and then if we give reverse option, flame graph is not generated in normal. So I feel better to have both for this feature.

Now both "calltree" and "backtrace" both work in same UI. I have fixed this.